### PR TITLE
removed embedded licenses from scripts 

### LINKED ISSUE #7088: GenericTsvReader.py and CreateCosmicGene.tsv have embedded licenses
Can these be removed? They appear to date from the Oncotator era...

### PR DESCRIPTION
**Pull request by haileypfox**
resolves #7088

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java [hunk 1]
```diff
@@ -99,8 +99,8 @@
  *   -I input.bam \
  *   -O output.g.vcf.gz \
  *   -ERC GVCF \
- *   -G Standard \
- *   -G AS_Standard
+ *   -G StandardAnnotation \
```
Comment (jonn-smith) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java:102]: Looks like some extra code was included in this PR.  Can you revert this change?

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/LeftAlignAndTrimVariants.java [hunk 2]
```diff
@@ -94,7 +94,7 @@
  *   -V input.vcf \
  *   -O output.vcf \
  *   --split-multi-allelics \
- *   --dont-trim-alleles
+ *   --dont-trim-alleles \
```
Comment (jonn-smith) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/LeftAlignAndTrimVariants.java:97]: Looks like some extra code was included in this PR. Can you revert this change?

**Review by jonn-smith — CHANGES_REQUESTED** [review]
Looks like some extra code was included in this PR.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java (hunk 1 above)
Comment (haileypfox) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java]: Okay Fixed!

**Review by jonn-smith — APPROVED** [review]
Looks good!

Feel free to merge when tests pass.
